doc/node-types-html/index.html.in

```diff
@@ -15,7 +15,229 @@
+	        function onThumbExampleClick(event){
+	            loadExampleImage($(this));
+	        }
+
+	        /* loads the correspondent image from the thumbnail */
+	        function loadExampleImage(_thumb)
+	        {
+	            /* removing old image reference */
+	            $('#example-image-container').empty();
+	            $("img[id^=thumb_]").each(function(){
+	                $(this).removeClass('example-thumb-active');
+	            });
+
+	            /* loading selected image */
+	            var imageURL = $(_thumb).data('large');
+
+	            // SVG Loading happens here 
+	            var image = $("<img></img");
+	            image.attr("src",imageURL);
+	            image.attr('id','example-image');
+	            image.data('code',$(_thumb).data('code'));
+
+	            /* adding the loaded image into the drag view */
+	            $("#example-image-container").append(image);
+
+	            /* marking the active thumb */
+	            $(_thumb).addClass('example-thumb-active');
+
+	            /* making the image draggable */
+	            image.draggable();
+
+	            /* removing existing copy feedback and restoring the copy button */
+	            if($('#example-info').has('#example-code-copied')){
+	                $('#example-code-copied').remove();
+	                $('#example-get-code').show();
+	            }
+	        }
+
+	        /* --------------------------------------------------------
+	         * COPY EXAMPLE CODE
+	         * Implement the algorithm to read the example
+	         * code here. The var code already stores url 
+	         * when creating the Entry in the main loop
+	         * --------------------------------------------------------
+	         */
+	        function copyExampleCode(event){
+	            var x = location.protocol;
+	            var y = /^file?:/i;
+	            if(y.test(x)){
+	                alert("Error: Code cannot be copied! \nPlease run Cheat Sheet from server");
+	            }
+	            else{
+	                var text = document.getElementById('example-info-code');
+	                selecttext(text);
+	                copycode();
+	                $(this).fadeOut(100,showCopiedFeedback);
+	            }
+	        }
+	        
+	        function selecttext(element){
+	            var range = document.createRange();// create new range object
+	            range.selectNodeContents(element);// set range to encompass desired element text
+	            var selection = window.getSelection(); // get Selection object
+	            selection.removeAllRanges(); // unselect any user selected text (if any)
+	            selection.addRange(range); // add range to Selection object to select it
+	        }
+	        
+	        function copycode(){
+	            var copycheck; // var to check whether execCommand successfully executed
+	            try{
+	                copycheck=document.execCommand('copy'); // run command to copy selected text
```

Comment:
Split this function like the others have done in the code:
From: 
```copycheck=document.execCommand('copy')```

to

```copycheck = document.execCommand('copy')```

Change the name of the var from copycheck to copyCheck